src/dropdown/dropdown.js

```diff
@@ -156,4 +177,54 @@ angular.module('ui.bootstrap.dropdown', [])
       });
     }
   };
-});
+})
+
+.directive('dropdownMenu', ['$document', '$timeout', function ($document, $timeout) {
+  return {
+    restrict: 'AC',
+    require: '?^dropdown',
+    link: function (scope, element, attrs, dropdownCtrl) {
+      if (!dropdownCtrl) {
+        return;
+      }
+      dropdownCtrl.register(onOpen);
+      function onOpen(coords) {
+        if (!coords) {
+          return;
+        }
+        // Hide it before moving it in to the right place
+        element.css('display', 'none');
+        // Needs to be done in a timeout, otherwise it is not visible and thus has no width/height
+        $timeout(function () {
```

Comment:
This should support usage of $animate I think, in case the user wants to animate in the menu